oswrap/oswrap.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Why do some files have a different license format and 2016 date?

Reply:
they were copy/pasted from somewhere else

Reply:
Can you note that somewhere? @adjackura may know if we have specific guidelines for this. But it's useful to know where they were copied from to explain 1) why the style and formatting are different and 2) where to look in case there's a bug that might have been already fixed in a different location.

Reply:
I only needed one function from them for now so I inlined that function and put a link to where I copied it from